account_check_printing_report_base/models/res_company.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+# (http://www.eficent.com)
+# Copyright 2016 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+REPORTS_DEFINED = [
+    ("account_check_printing_report_base.action_report_check_base", "Print Check Base"),
+    (
+        "account_check_printing_report_base.action_report_check_base_a4",
+        "Print Check Base A4",
+    ),
+]
+REPORTS_DEFINED_ONDELETE = {
+    "account_check_printing_report_base.action_report_check_base": "set default",
+    "account_check_printing_report_base.action_report_check_base_a4": "set default",
+}
```

Comment:
Can you try this to avoid extend two constansts:
```suggestion
REPORTS_DEFINED_ONDELETE = {r[0]: "set default" for r in REPORTS_DEFINED}
```